extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -0,0 +1,46 @@
+import {Layout_Enum} from '#core/dom/layout';
+
+import {Services} from '#service';
+
+import {CSS} from '../../../build/amp-story-audio-sticker-0.1.css';
+import {
+  Action,
+  StateProperty,
+} from '../../amp-story/1.0/amp-story-store-service';
+
+const TAG = 'amp-story-audio-sticker';
+
+export class AmpStoryAudioSticker extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?../../../extensions/amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+  }
+
+  /** @override */
+  buildCallback() {
```

Comment:
Nit: I think this may go in `layoutCallback` because it's just a click listener (aka: after the layout has resolved). @gmajoulet can confirm.